tests/test_bootstrap.py

```diff
@@ -100,18 +100,29 @@ def test_build_dist_dirs(self):
             bs.get_common_dir().endswith("pythonforandroid/bootstraps/common")
         )
 
-    def test_list_bootstraps(self):
+    @mock.patch("os.listdir")
+    def test_list_bootstraps(self, mock_listdir):
         """A test which will initialize a bootstrap and will check if the
         method :meth:`~pythonforandroid.bootstrap.Bootstrap.list_bootstraps`
         returns the expected values, which should be: `empty", `service_only`,
         `webview` and `sdl2`
         """
-        expected_bootstraps = {"empty", "service_only", "webview", "sdl2"}
-        set_of_bootstraps = set(Bootstrap().list_bootstraps())
+
+        mock_listdir.return_value = \
+            ["empty", "sdl2", "service_only", "webview"]
+        expected_bootstraps = sorted(
+            ["empty", "service_only", "webview", "sdl2"]
+        )
```

Comment:
minor suggestion to improve the readability of this block:
```python
listdir_ret = ["empty", "service_only", "webview", "sdl2"]
mock_listdir.return_value = listdir_ret
expected_bootstraps = sorted(listdir_ret)
```
That way we make it clear that we're talking about the same list once sorted once not. Because here you somehow shuffled your lists and at first sight we don't see they're the same or why they're shuffled.